FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -28,11 +24,13 @@
  */
 #define FBSnapshotVerifyViewWithOptions(view__, identifier__, suffixes__) \
 { \
+NSString *envReferenceImageDirectory = [NSProcessInfo processInfo].environment[@"FB_REFERENCE_IMAGE_DIR"]; \
```

Comment:
I think Swift will support #if/#else preprocessor macros. Is there a way to use the preprocessor flag by default and fall back to the environment variable when the flag is undefined?